.binder/requirements.txt

```diff
@@ -0,0 +1,24 @@
+Cython>=0.29.13
```

Comment:
Could we revisit my earlier suggestion for the requirements file - https://github.com/scikit-image/scikit-image/pull/4340#discussion_r357905639 ?

Reply:
So I partially intregrated your suggestion :-), in the sense that I used two existing files but for the doc-specific requirements (scikit-learn etc.) I did not want to use all the requirements of `docs.txt` since we don't need sphinx copy-button etc. Another possibility would be to separate docs.txt in two files, one  `docs_build.txt` for the build itself and one `docs_examples.txt` for running the examples only.

Reply:
I would just use the same docs requirements, I do not see overshooting to be a major issue. The current version is also fine. Thank you, Emma!